slang.h

```diff
@@ -2251,13 +2253,17 @@ extern "C"
         // The register space offset for the sub-elements that occupies register spaces.
         SLANG_PARAMETER_CATEGORY_SUB_ELEMENT_REGISTER_SPACE,
 
+        // The input_attachment_index subpass occupancy tracker
+        SLANG_PARAMETER_CATEGORY_SUBPASS,
+
         //
-        SLANG_PARAMETER_CATEGORY_COUNT,
+        SLANG_PARAMETER_CATEGORY_COUNT_REV2,
```

Comment:
Is this necessary? What happens if we just update the value of `SLANG_PARAMETER_CATEGORY_COUNT`?

Reply:
My preference is we don't introduce this complexity unless absolutely necessary.

Reply:
`SLANG_PARAMETER_CATEGORY_COUNT` changing value broke a Falcor test. 

I have no preference on solution if another also solves the issue.

Reply:
Maybe keep the actual count named as count, and the old one named as COUNT_V1.

Reply:
I will check if this works.